lib/ClangImporter/ImportName.cpp

```diff
@@ -1709,11 +1709,12 @@ ImportedName NameImporter::importNameImpl(const clang::NamedDecl *D,
   case clang::DeclarationName::CXXOperatorName: {
     auto op = D->getDeclName().getCXXOverloadedOperator();
     auto functionDecl = dyn_cast<clang::FunctionDecl>(D);
-    if (!functionDecl) {
-      // This can happen for example for templated operators functions.
-      // We don't support those, yet.
+
+    if (auto functionTemplate = dyn_cast<clang::FunctionTemplateDecl>(D))
+      functionDecl = functionTemplate->getAsFunction();
```

Comment:
Since `VisitFunctionTemplateDecl` already calls this logic with `decl->getAsFunction()`, I think this line won't be executed. Are you planning to tweak `VisitFunctionTemplateDecl` to import `decl` as is, not as `decl->getAsFunction()`?

Reply:
Yeah, I don't think this path is taken yet. This is needed for lazy member loading, when we import the name of every `NamedDecl` member. 

I'm not going to remove `decl->getAsFunction` yet as it is still needed for some non-operator cases. But that might be a good follow up.